bmeg/bmeg
# correct state inconsistency on path change; 	lock material-ui version at 1.0.0-beta.6

This PR:
* simplifies state management ( one less reducer 'search' )
* corrects bug : query string state was erased when changing vertex

To test:
* new session /cohort/Project
* select `Project.name:("TCGA-ACC" "TCGA-HNSC" )`
* navigate to vertex Gene
* navigate back to to vertex Project
* verify that the query reverts to `Project.name:("TCGA-ACC" "TCGA-HNSC" )`
